tsfresh/feature_extraction/settings.py

```diff
@@ -152,6 +152,8 @@ def __init__(self):
             "lempel_ziv_complexity": [{"bins": x} for x in [2, 3, 5, 10, 100]],
             "fourier_entropy":  [{"bins": x} for x in [2, 3, 5, 10, 100]],
             "permutation_entropy":  [{"tau": 1, "dimension": x} for x in [3, 4, 5, 6, 7]],
+                        "matrix_profile": [{"sample_pct": 1, "threshold": 0.98, "feature": f}
```

Comment:
Currently, as `window` is not included, those kwargs are never used. Should we remove them?

Reply:
@nils-braun per the earlier comment, I'll remove "sample_pct" and leave "threshold" for the time being.